classes/assets/JavascriptManager.php

```diff
@@ -73,12 +74,15 @@ public function unregisterById($idToRemove)
         }
     }
 
-    protected function add($id, $fullPath, $position, $priority, $inline, $attribute, $server)
+    protected function add($id, $fullPath, $position, $priority, $inline, $attribute, $server, $version)
     {
         $priority = is_int($priority) ? $priority : self::DEFAULT_PRIORITY;
         $position = $this->getSanitizedPosition($position);
         $attribute = $this->getSanitizedAttribute($attribute);
 
+        //../javascript.js?{version}
+        $fullPath = ($version)? $fullPath.'?'.$version : $fullPath;
```

Comment:
```suggestion
        $fullPath = $version ? $fullPath.'?'.$version : $fullPath;
```

make sure that if someone is using`//domain.com?stylesheet=1` using `external` type, we still can use our own versioning

Reply:
@kpodemski yes I've tested it with `external` type. I uploaded changes on:
https://ps1773.binshops.com/
as you see the following link works fine:
https://ps1773.binshops.com/modules/ps_imageslider/css/homeslider.css?v1.2